src/client/App.jsx

```diff
@@ -1,10 +1,144 @@
 import './App.css';
+import {useState} from 'react'
+import Form from './components/form';
+import Input from './components/input';
+
 
 const apiUrl = 'http://localhost:4000';
 
+
 function App() {
+  const [user, setUser] = useState({username: '', password: ''})
+  const [movieParams, setMovieParams] = useState({title: '', description: '', runtimeMins: ''})
+  const [movies, setMovies] = useState([])
+
+  
+  const handleRegister = (event) => {
+    event.preventDefault()
+
+    const options = {
+      method: "POST",
+      headers: {
+        "content-type": "application/json"
+      },
+      body: JSON.stringify(user)
+    }
+
+    fetch('http://localhost:4000/user/register', options)
+    .then(response => response.json())
+   
+
+  
+  }
+
+
+  const handleLogin = (event) => {
+    event.preventDefault()
+
+    const options = {
+      method: "POST",
+      headers: {
+        "content-type": "application/json"
+      },
+      body: JSON.stringify(user)
+    }
+
+    fetch('http://localhost:4000/user/login', options)
+    .then(response => response.json())
+    .then(data => {
+      const response = data.token
+
+      // Save Token
+      localStorage.setItem('Token', response)
```

Comment:
You could also save the token as part of the register function - this way when a user registers they wouldn't have to log in afterwards